api/v1alpha1/mirrorpeer_types.go

```diff
@@ -69,6 +69,14 @@ type MirrorPeerSpec struct {
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default=false
 	ManageS3 bool `json:"manageS3,omitempty"`
+
+	// OverlappingCIDR should be set to true if the peer clusters
+	// have overlapping Pod or Service CIDR. This will enable storage
+	// clusters to use submariner globalnet. Enabling this will cause
+	// storage service disruption while network is being reconfigured.
```

Comment:
Are there any repercussions if this setting is flipped later on ? Do we need add a webhook check to disallow changing it later on ?

Reply:
Do we want to have this setting here  if there would be service  disruption and may impact the working of other odf components ?